lib/widgets/password_field.dart

```diff
@@ -13,7 +17,7 @@ class _PasswordFieldState extends State<PasswordField> {
   @override
   void initState() {
     super.initState();
-    _passwordVisible = false;
+    _passwordVisible = true;
```

Comment:
🔴 
the issue didn't state the need to change the initial state of password visibility. Don't deviate from the issue requirements. If you have any change suggestion, discuss with me in the issue before making the changes.